examples/connectivity/plot_dict_learning_resting_state_shuffle.py

```diff
@@ -0,0 +1,84 @@
+"""
+Group analysis of resting-state fMRI with dictionary learning: DictLearning
+=====================================================
+
+An example applying dictionary learning to resting-state data. This example
+applies it to 20 subjects of the ADHD200 datasets.
+
+Dictionary learning is a sparsity based decomposition method
+for extracting spatial maps.
+
+    * Gael Varoquaux et al.
+    Multi-subject dictionary learning to segment an atlas of brain
+    spontaneous activity
+    Information Processing in Medical Imaging, 2011, pp. 562-573,
+    Lecture Notes in Computer Science
+
+Pre-prints for paper is available on hal
+https://hal.inria.fr/inria-00588898/en/
+"""
+import numpy as np
+
+### Load ADHD rest dataset ####################################################
+from nilearn import datasets
+# For linear assignment (should be moved in non user space...)
+
+hcp_dataset = datasets.fetch_hcp_rest(data_dir='/volatile3',
```

Comment:
Is the plan to add fetch_hcp_rest to nilearn.datasets? Maybe you didn't mean to git add this example.